project/Dependencies.scala

```diff
@@ -1,6 +1,9 @@
 import sbt._
 
 object Dependencies {
+  lazy val scala212 = "2.12.13"
```

Comment:
Nit: doesn't need to be lazy

Reply:
(it'd need if the initialization order was fishy and you risked the value being referenced before initialization, but here it's rather clear AFAICS)